src/shared/tpm2-util.h

```diff
@@ -67,9 +73,14 @@ Tpm2Context *tpm2_context_unref(Tpm2Context *context);
 DEFINE_TRIVIAL_CLEANUP_FUNC(Tpm2Context*, tpm2_context_unref);
 #define _cleanup_tpm2_context_ _cleanup_(tpm2_context_unrefp)
 
+typedef enum TPM2HandleFlags {
+        TPM2_HANDLE_FLAGS_NOFLUSH = 1 << 0,
+} TPM2HandleFlags;
+
 typedef struct {
         Tpm2Context *tpm2_context;
         ESYS_TR esys_handle;
+        TPM2HandleFlags flags;
```

Comment:
just wondering, but why a flags field? why not just make this a bool?

Reply:
I thought about that initially, but since everything in this is new, I figured other uses might emerge and id lay in the plumbing to avoid having to update all callers and users at a later date.

Reply:
changed to bool